help.go

```diff
@@ -44,7 +44,8 @@ func (e *Executor) printTasks(listAll bool) {
 	// Format in tab-separated columns with a tab stop of 8.
 	w := tabwriter.NewWriter(e.Stdout, 0, 8, 0, '\t', 0)
 	for _, task := range tasks {
-		fmt.Fprintf(w, "* %s: \t%s\n", task.Name(), task.Desc)
+		taskNames := append([]string{task.Name()}, task.Aliases...)
+		fmt.Fprintf(w, "* %s: \t%s\n", strings.Join(taskNames, "|"), task.Desc)
```

Comment:
I believe this may possible break the auto-completion scripts that use `--list` to parse the list of available tasks.

Because of this, I think we may need to move the somewhere after. Not sure about the format, but maybe something like this?:

```bash
task: Available tasks for this project:
* foo:                prints foo (aliases: f)
* included:bar        prints bar (aliases: included:b, inc:b)
```

Reply:
Fine with this. Do we want to print them in a new column? or just inline next to the description. A column might make this info easier to parse.

Also, same question as `--summary` flag below - Should we colour the aliases?

Reply:
I end up changing that myself, taking your suggestion of printing in a separate column, and colored.